src/internal/provider/platform.go

```diff
@@ -0,0 +1,7 @@
+package provider
+
+// Platform represents a platform that a provider supports.
```

Comment:
Nitpick: Maybe this should live in v1api as it's only used in there?

Reply:
I agree, I'll move it